hadoop-hdds/dev-support/checkstyle/suppressions.xml

```diff
@@ -18,4 +18,5 @@
 
 <suppressions>
   <suppress checks="JavadocPackage" files="[\\/]src[\\/]test[\\/].*"/>
+  <suppress checks="JavadocPackage" files="src/main/java/org/apache/hadoop/io/erasurecode/rawcoder"/>
```

Comment:
BTW what about dropping `JavadocPackage` rule? I believe it's adopted from Hadoop code style, but it doesn't seem to make much sense.

Reply:
Removing the rule will allow to remove useless (in most cases) package-info files.